README.md

```diff
@@ -1 +1,31 @@
-# cram
\ No newline at end of file
+### Welcome to cram2!
+=============
+
+### What is CRAM
+
+CRAM is a toolbox for designing, implementing and deploying software on autonomous robots. The framework provides various tools and libraries for aiding in robot software development as well as geometric reasoning and fast simulation mechanisms to develop cognition-enabled control programs that achieve high lev- els of robot autonomy. CRAM also provides tools for introspection that enable the robots to reason about their past executions and improve by autonomously optimizing their plans.
+
+The core packages of CRAM are implemented in Common Lisp (with a little bit of C/C++) with support to the ROS middleware infrastructure.
+
+### Installation
+
+*  If you wish to install cram please follow the instruction here: [cram-installation guide](http://cram-system.org/installation).
+
+
+### Directory
+* **cram_3d_world** Bullet physics engine-based and OpenGl offscreen rendering-based reasoning mechanisms 
+and lightweight simulation of PR2 robot.
+* **cram_3rdparty** 3rd party Lisp liabraries wrapped into ROS packages.
+* **cram_boxy** Hardware interface for using Boxy robot in CRAM.
+* **cram_common** Common libraries building on top of CRAM core. Including costmap and implementations.
+* **cram_core** Tools and Interfaces for writing Cognition-Enabled Reactive Concurrent Plans.
+* **cram_jason_prolog** ROS JSON Prolog client implementation in Lisp: sending Prolog queries in JSON format over ROS. 
+* **cram_knowrob** Libraries for using knowrob in Lisp
+* **cram_pr2** Hardware interface for using PR2 robot in CRAM
```

Comment:
Lightweight simulation and projection module are the same thing. Could you get rid of one of them, otherwise people can start wondering what is the simulation and what is projection and what is the difference, whereas there is no difference.

Reply:
Done i thought they are different things.